extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,29 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!--
+  Forced to use !important here as it makes the code easy to understand, but does not align with design principles.
+  A solution could be added using CSS, but for some reason the styles defined in this file seem to be overriden by Guppy.
+  Only way to fix this is to override the specificity using !important and force the component to take the styles from here
+  and not the styles defined by Guppy. It seems that even customizing the guppy active or guppy inactive classes has no
+  effect on the styles of the textbox. Again, !important is required to break the specificity for height.
+-->
```

Comment:
I'm still missing the description what exactly is this whole part fixing. I mean something like "these changes are needed in order to fix line breaks in text rendered by Guppy".

Reply:
I've refactored the comments to explain them better.